### ISSUE TITLE
UHD-404VW Seamless Matrix Support

### ISSUE THREAD

**Issue by ukiews** [bitfocus/companion-module-orei-matrix#2]
Will this work this unit?:
OREI Professional 4K 4X4 HDMI Matrix Video Wall, Seamless Switching HDCP 2.2 UltraHD

https://a.co/d/gqpgmC6

**Comment by jrviz** [comment]
No, I don't think it will. The switch you show is controlled by a PC app or a browser. There is no telnet connection or command set in the specification or manual.

**Comment by ukiews** [comment]
Is this supported:
https://www.orei.com/products/uhd44ex230k-4-in-4-hdmi-matrix-switch-extender

**Comment by ukiews** [comment]
Or perhaps you have a list of devices that are supported?
